src/seedu/addressbook/ui/TextUi.java

```diff
@@ -83,37 +87,40 @@ public String getUserCommand() {
             fullInputLine = in.nextLine();
         }
 
-        showToUser("[Command entered:" + fullInputLine + "]");
+        showToUser(COMMAND_ENTER, "[Command entered:" + fullInputLine + "]");
         return fullInputLine;
     }
 
 
     public void showWelcomeMessage(String version, String storageFilePath) {
         String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
-        showToUser(
-                DIVIDER,
-                DIVIDER,
+        showToUser(WELCOME_MESSAGE,
                 MESSAGE_WELCOME,
                 version,
                 MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
-                storageFileInfo,
-                DIVIDER);
+                storageFileInfo);
```

Comment:
Some of these could still be pushed to the formatter to format the welcome message.  A little too much formatting mechanism here.